scripts/run_clang_tidy_format.sh

```diff
@@ -4,8 +4,8 @@ if [ ! -d sdk ] ; then
 	echo Please run this script from the root of the cheriot-rtos repository.
 	exit 1
 fi
-CLANG_TIDY=/cheriot-tools/bin/clang-tidy
-CLANG_FORMAT=/cheriot-tools/bin/clang-format
+CLANG_TIDY=/llvm/build-linux/bin/clang-tidy
```

Comment:
I think this was committed accidentally?

Reply:
Oops, fixed.